### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by andrew-kuzmin**
Please review and approve if no objections. I'm creating a new branch for the registration process.

### CODE REVIEW

File: src/main/java/com/phoenixoft/teambalanceapp/util/JwtUtil.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
 @Service
 public class JwtUtil {
 
-    private String SECRET_KEY = "secret";
+    private String SECRET_KEY = "tb_secret";
```
Comment (JackKarichkovskiy) [inline src/main/java/com/phoenixoft/teambalanceapp/util/JwtUtil.java]: I think such things should be extracted to the config file. Locally: could be just constant value. Prod: environment variable.

File: src/main/resources/db/changelog/changes/insert_credentials_data.sql [hunk 2]
```diff
@@ -1,2 +1,2 @@
-insert into public.tbl_user_credentials (id, username, password) values (1, 'andriy', 'root');
-insert into public.tbl_user_credentials (id, username, password) values (2, 'eugen', 'admin');
\ No newline at end of file
+insert into public.tbl_user_credentials (id, username, password) values (1, 'andriy', 'andriy');
+insert into public.tbl_user_credentials (id, username, password) values (2, 'eugen', 'eugen');
```
Comment (JackKarichkovskiy) [inline src/main/resources/db/changelog/changes/insert_credentials_data.sql]: Credentials is a sensitive information. So you can extract it to the separate sql file for future local use. But we can't use it in Liquibase like that for security purposes.

Comment (andrew-kuzmin) [reply src/main/resources/db/changelog/changes/insert_credentials_data.sql]: I agree with you. I've made that changes just for clarity in the debug process and not to remember a combination of user-password. We will store it somewhere else in the future.

File: src/main/java/com/phoenixoft/teambalanceapp/util/JwtUtil.java (hunk 1 above)
Comment (andrew-kuzmin) [reply src/main/java/com/phoenixoft/teambalanceapp/util/JwtUtil.java]: I suggest moving this to the config file when we will create separate run profiles.

Comment (JackKarichkovskiy) [reply src/main/java/com/phoenixoft/teambalanceapp/util/JwtUtil.java]: Good idea!

File: src/main/resources/db/changelog/changes/insert_credentials_data.sql (hunk 2 above)
Comment (JackKarichkovskiy) [reply src/main/resources/db/changelog/changes/insert_credentials_data.sql]: Cool!

**Pull request closed without merging**